### PR TITLE
Something like Ctrl w + T in vim implemented ##panels

### PR DESCRIPTION
**Pull request by Vane11ope**
and Fixed a few bugs

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -4889,7 +4889,7 @@ static RPanels *get_cur_panels(RPanelsRoot *panels_root) {
 	return get_panels (panels_root, panels_root->cur_panels);
 }
 
-static bool handle_tab(RCore *core) {
+static bool handle_tab(RCore *core, bool *del_tab) {
```
Comment (radare) [inline libr/core/panels.c]: Not a big fan of this new signature. It doesnt escales and its confusing to use. Im fine for merging i guess you’ll be diing more refactors over time

File: libr/core/panels.c [hunk 2]
```diff
@@ -4970,6 +4970,38 @@ static void handle_tab_new(RCore *core) {
...
+
+	RPanelsRoot *root = core->panels_root;
+	if (root->n_panels >= PANEL_NUM_LIMIT) {
+		return;
+	}
+
+	RPanel *cur = getCurPanel (panels);
+
+	root->n_panels++;
+	root->panels[root->n_panels - 1] = panels_new (core);
```
Comment (radare) [inline libr/core/panels.c]: Chk for nul

File: libr/core/panels.c [hunk 3]
```diff
@@ -5374,8 +5406,15 @@ static int panels_process(RCore *core, RPanels **r_panels, bool *force_quit) {
 		}
 		break;
 	case 't':
-		if (handle_tab (core)) {
-			return false;
+		{
+			bool del_tab = false;
+			if (handle_tab (core, &del_tab)) {
+				if (del_tab) {
+					goto exit;
```
Comment (radare) [inline libr/core/panels.c]: ELse after goto. No pls

File: libr/core/panels.c (hunk 1 above)
Comment (Vane11ope) [reply libr/core/panels.c]: yup exactly. i will be refactoring this later on but currently '-' keys in tab emits a bug anyway without this.

File: libr/core/panels.c (hunk 3 above)
Comment (Vane11ope) [reply libr/core/panels.c]: ah ok

File: libr/core/panels.c (hunk 2 above)
Comment (Vane11ope) [reply libr/core/panels.c]: fair enough. a few more points just occurred to me to fix in fact :)

**Comment by Vane11ope** [comment]
this indeed contains bunch of buggy codes, let me quickly fix them.

**Comment by Vane11ope** [comment]
thought enough for now but hang on yet

**Comment by Vane11ope** [comment]
yeah in fact didn't come up with cleaner solution yet, will do refactoring soon enough.

**Pull request merged**